deliver/lib/deliver/upload_metadata.rb

```diff
@@ -184,7 +184,7 @@ def load_from_filesystem(options)
           next unless File.exist?(path)
 
           UI.message("Loading '#{path}'...")
-          options[key] ||= {}
+          options[key] = {}
```

Comment:
My Fastfile contains the following call to `deliver()`

```
deliver(
      username: "#{ENV['FASTLANE_APPLE_ID']}",
      name: "#{ENV['FASTLANE_APP_NAME']}",
      team_id: "#{ENV['FASTLANE_ITC_TEAM_ID']}", 
      app_identifier: "#{ENV['FASTLANE_APP_IDENTIFIER']}", 
      force: true,
      skip_screenshots: true
    )
```
So I'm passing a specific `name:` value to `deliver()`. Internally, Fastlane is setting `options['name'] = {the value I passed as name:}`.
 
I also have the following folders/files:
```
metadata/
  en-US/
    description.txt
    keywords.txt
    name.txt
    ...
```

When Fastlane calls `load_from_filesystem(options)` (in upload_metadata.rb), it passes in the `options` hash and attempts to augment `options` with data read from the ./metadata files. 

However, as it reads the ./metadata files, it does the following:
```
options[key][language] ||= File.read(path)
```

It's this line that fails when `key` == "name" because `options['name']` is already set to a string, as described above. Since `options['name']` is a string, trying to assign `options['name']['en-US']` fails.